README.md

```diff
@@ -0,0 +1,37 @@
+# xrally-kubernetes
+
+xRally plugins for `Kubernetes <https://kubernetes.io/>`_ platform.
+
+## Status ot this package
+
+Work in progress. The active phase of development.
+
+## Getting started
+
+First of all, you need to create rally env for kubernetes. Use spec from
+`samples/platforms/cert-spec.yaml` to create env:
```

Comment:
NIT: let's give more details here and say that "First of all, you need to create rally env for Kubernetes. There are two main ways to communicate to Kubernetes cluster - specifying auth-token or certifications. Choose what is suitable for your case and use one of the following samples."